src/services/storage/drivers/relational/migrations/20210210214139_urlsInformation.ts

```diff
@@ -0,0 +1,25 @@
+import * as Knex from 'knex'
+
+export async function up(knex: Knex): Promise<void> {
+	await knex.raw('PRAGMA foreign_keys = ON')
+	const urlAlterUnique = knex.schema.alterTable('urls', function (table) {
+		table.text('id').primary().alter()
+	})
+	const urlInfo = knex.schema.createTable('url_information', function (table) {
+		table.text('url_id')
+		table.foreign('url_id').references('id').inTable('urls').onDelete('CASCADE')
+		table.string('ip')
+		table.integer('url_visit_count')
+		table.integer('info_visit_count')
+		table.string('region')
+		table.dateTime('last_use')
```

Comment:
The convention is using naming like `created_at` and `updated_at`, so according to that, I think you should make it `last_used` here (and in src/services/storage/types/url.ts on UrlWithInformation).

Reply:
done